src/Sentry/DelegatingMetricAggregator.cs

```diff
@@ -0,0 +1,16 @@
+namespace Sentry;
+
+internal class DelegatingMetricAggregator(IMetricAggregator innerAggregator) : IMetricAggregator
+{
+    public void Increment(string key, double value = 1, MeasurementUnit? unit = null, IDictionary<string, string>? tags = null,
```

Comment:
Should probably call this `Incr` for consistency with the other SDKs.

Reply:
.NET folks spell things out and since no one writes C# without an IDE even if we documented incorrectly Incr, they'd find Increment

I'd say it's fine to deviate here but don't feel strongly on this

Reply:
> .NET folks spell things out 

In other words: like like Objective-C where it defaults to complete names instead of abbreviation. That's why everything is so verbose :)

Reply:
This one seems like the odd one out in the original Python implementation as well... all the other metrics just use the metric type name as the method that is used to emit the metric (e.g. `Set`, `Distribution`, `Gauge`). For some reason `Incr` is used instead of `Counter` though. Would it makes sense to rename `Incr` to `Counter`? 

If this is a nouns and verbs thing, it's also not consistent at the moment. Something like `AddCounter`, `AddSet`, `AddGauge` etc. might work for consistent verbs names.